conf/configuration.go

```diff
@@ -34,6 +34,7 @@ type configOptions struct {
 	ImageCacheSize          string
 	AutoImportPlaylists     bool
 	PlaylistsPath           string
+	AutoTranscodeDownload   bool
```

Comment:
I think this should be called `DefaultDownloadTranscoding` (or similar), as it can be overridden in the UI.